integration-test-suite/cds-toolkit-test-framework/src/main/groovy/org/wso2/cds/test/framework/constant/AUConstants.groovy

```diff
@@ -617,6 +617,9 @@ class AUConstants extends OBConstants {
     public static final String CONTENT_TYPE_APPLICATION_SCIM_JSON = "application/scim+json"
     public static final String SUBSCRIBER_ROLE = "Internal/subscriber"
     public static final String PUBLISHER_ROLE = "Internal/publisher"
+    public static final String TEST_SMS_CLIENT_ID = "AC34f40df03e20fb6498b3fcee256ebd3b"
+    public static final String TEST_SMS_CLIENT_SECRET = "5fad3edc88ac553b2abf785b52c81adb"
+    public static final String TEST_SMS_URL = "https://api.twilio.com/2010-04-01/Accounts/AC34f40df03e20fb6498b3fcee256ebd3b/SMS/Messages.json"
```

Comment:
Shall we remove these since we are using backup codes for the SMSAuthentication?